first PR with new MVTX decoding code

ycorrales opened the pull request.

mpurschke commented:
We are forbidden to accept a commit that states
// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
// All rights not expressly granted are reserved.
//

We need to change this, sorry (we went through this with some vendor's header files a while ago)

ycorrales commented:
I have removed the copyright, actually it was the right action from the beginning since I have modified all classes to adapt them to MVTX requirements...

mpurschke reviewed (approved):
ok...
